base/abstractarray.jl

```diff
@@ -337,12 +337,18 @@ end
 
 zero{T}(x::AbstractArray{T}) = fill!(similar(x), zero(T))
 
-## iteration support for arrays as ranges ##
+## iteration support for arrays ##
+macro _inline_meta()
+    Expr(:meta, :inline)
```

Comment:
A comment here about `pushmeta!` not yet being available in the bootstrap might be handy. Otherwise
there's a (small) risk we might see usages of `Base.@_inline_meta` popping up in package code from folks who learn by browsing through the source.